src/main/java/ch/cern/eam/wshub/core/services/equipment/impl/LocationServiceImpl.java

```diff
@@ -248,4 +240,22 @@ private LocationParentHierarchy getLocationParentHierarchy(InforContext context,
 
 		return locationParentHierarchy;
 	}
+
+	private USERDEFINEDAREA getInforCustomFields(
+		InforContext context,
+		Location locationParam,
+		net.datastream.schemas.mp_entities.location_001.Location locationInfor)
+			throws InforException {
+
+		String classCode;
+		String targetClassCode = locationParam.getClassCode();
+		net.datastream.schemas.mp_fields.CLASSID_Type sourceClassId = locationInfor.getCLASSID();
```

Comment:
NullPointerException
Besides, I should not need to pass a specific location to be able to read custom fields